src/App/Cart/Service/CartCalculationService.php

```diff
@@ -80,4 +87,19 @@ protected function hasDeliveryOptions(PdkCart $cart): bool
 
         return $anyItemIsDeliverable && ! $deliveryOptionsDisabled;
     }
+
+    /**
+     * @param  \MyParcelNL\Pdk\App\Cart\Model\PdkCart $cart
+     * @param  string                                 $packageType
+     *
+     * @return bool
+     */
+    private function isWeightUnderPackageTypeLimit(PdkCart $cart, string $packageType): bool
+    {
+        return $cart->lines->getTotalWeight() <= Arr::get(
+                Pdk::get('packageTypeWeightLimits'),
+                $packageType,
+                20000
```

Comment:
Geen random-maar-toch-logische waarde gebruiken, doe dan `INF`. En zet dit `Arr::get` brok even in een variabele, dit is slecht leesbaar.